runtime/src/main/java/org/corfudb/runtime/collections/CorfuCounter.java

```diff
@@ -0,0 +1,26 @@
+package org.corfudb.runtime.collections;
```

Comment:
Doesn't seem to be required. Please see comment below.

Reply:
Please see response below.